.github/scripts/validate-version.sh

```diff
@@ -3,21 +3,40 @@
 validate_version() {
   local version="$1"
   if [[ ! "$version" =~ ^[0-9]+\.[0-9]+\.[0-9]+(-pre.*)?$ ]]; then
-    echo "::error::Invalid version '$version' in $(realpath VERSION). Expected: X.Y.Z or X.Y.Z-pre*"
+    echo "::error::Invalid version '$version' in $(realpath "$version_file"). Expected: X.Y.Z or X.Y.Z-pre*"
```

Comment:
Isn't this a manual process? When was a non-valid version released that this validation is needed? I must admit I find this odd... but it might just be me...

Reply:
Yes it is manual, but people can make mistakes. Especially when making a pre-release, which does not require a PR review for a version release.